From: Michel Normand <normand@linux.vnet.ibm.com>
Subject: mozjs38 missing python before milestone call
Date: Fri, 06 Nov 2015 14:23:46 +0100

Add PYTHON setting before call to milestone to avoid errors like:
===
[   44s] checking for perl... /usr/bin/perl
[   44s] ./configure: line 1665: ./../../python/mozbuild/mozbuild/milestone.py: Permission denied
[   44s] ./configure: line 1666: ./../../python/mozbuild/mozbuild/milestone.py: Permission denied
[   44s] ./configure: line 1667: ./../../python/mozbuild/mozbuild/milestone.py: Permission denied
[   44s] checking for gcc... gcc
===

Signed-off-by: Michel Normand <normand@linux.vnet.ibm.com>
---
 js/src/configure.in |    5 +++++
 1 file changed, 5 insertions(+)

Index: mozjs-38.0.0/js/src/configure.in
===================================================================
--- mozjs-38.0.0.orig/js/src/configure.in
+++ mozjs-38.0.0/js/src/configure.in
@@ -140,6 +140,11 @@ if test -z "$PERL" -o "$PERL" = ":"; the
     AC_MSG_ERROR([perl not found in \$PATH])
 fi
 
+dnl Check for python -- needed for MOZILLA_VERSION
+MOZ_PATH_PROGS(PYTHON, $PYTHON python )
+if test -z "$PYTHON" -o "$PYTHON" = ":"; then
+    AC_MSG_ERROR([python not found in \$PATH])
+fi
 MOZ_ARG_ENABLE_BOOL(shared-js,
 [  --disable-shared-js
                           Do not create a shared library.],
